frontend/src/components/IstioWizards/WizardLabels.tsx

```diff
@@ -39,6 +39,15 @@ const clearButtonStyle = kialiStyle({
   marginLeft: '0.5rem'
 });
 
+const alertStyle = kialiStyle({
+  marginTop: '1rem',
+  $nest: {
+    '& .pf-v5-c-alert__title': {
+      marginTop: 0
+    }
+  }
+})
+
```

Comment:
The $nest part is not needed in the WizardLabels component, only the marginTop to `1rem`. 

```suggestion
const alertStyle = kialiStyle({
  marginTop: '1rem'
})

```